core/application/userWorkflow/UserService.ts

```diff
@@ -65,10 +56,7 @@ export class UserService {
     userId: string,
     userRepository: Repository<UserDetailsDTO>
   ): Promise<UserDetailsDTO> {
-    const userProfile = await userRepository.getItem(
-      `USER#${userId}`,
-      'PROFILE'
-    )
+    const userProfile = await userRepository.getItem(`USER#${userId}`, 'PROFILE')
     if (userProfile == null) {
```

Comment:
As we are following strict comparison in all places. I think you should use `===` here

Reply:
for some reason lint is not giving error

Reply:
updated in fb88d811b0c2627d95a87ab81566224901d34298